cmd/manager/main.go

```diff
@@ -54,6 +59,8 @@ var (
 	metricsHost               = "0.0.0.0"
 	metricsPort         int32 = 8383
 	operatorMetricsPort int32 = 8686
+	topologyPrefixes          = [...]string{"topology.kubernetes.io", "topology.block.csi.ibm.com"}
+	topologyEnabled     bool
```

Comment:
Why don't we set an initial value in contrast to other variables?

Reply:
cause it will have no meaning, its initiated before use

Reply:
Now, yes, it's initiated before we use it. Maintainance-wise, we need to remember there is an uninitialized variable here and IMO we should avoid remembering things like that.

Reply:
"uninitialized variable"?

Reply:
an uninitialized variable is a declared variable without a value.

Reply:
oh, I read unutilized..

Reply:
I removed it from being a public var completely